src/internal/operations/backup_integration_test.go

```diff
@@ -1755,7 +1755,7 @@ func runDriveIncrementalTest(
 					makeLocRef(container1))
 			},
 			itemsRead:    0,
-			itemsWritten: 7, // 2*2(data and meta of 2 files) + 3 (dirmeta of two moved folders and target)
+			itemsWritten: 0,
```

Comment:
this one seems a bit tricky and I'm not sure what the expectation is from the users' perspective. I could see them either expecting to see 2 files moved (since the folder changed) or no files moved (since the file content hasn't changed)

Reply:
If we see from a backup's perspective we did not backed up anything new. And it shows that we did not backup the same content again.